README.md

```diff
@@ -6,10 +6,10 @@
 
 ## 📦 Packages
 
-| Package                                  | Description                 | Version                                                                                                                    |
-|------------------------------------------|-----------------------------|----------------------------------------------------------------------------------------------------------------------------|
-| [algolia_helper](helper_dart)            | Algolia Helpers for Dart    | [![pub package](https://img.shields.io/pub/v/algolia_helper.svg)](https://pub.dev/packages/algolia_helper)                 |
-| [algolia_helper_flutter](helper_flutter) | Algolia Helpers for Flutter | [![pub package](https://img.shields.io/pub/v/algolia_helper_flutter.svg)](https://pub.dev/packages/algolia_helper_flutter) |
+| Package                                  | Description                 | Version                | Release stage                                                                                                           |
+|------------------------------------------|-----------------------------|-----------------------------|----------------------------------------------------------------------------------------------------------------------------|
+| [algolia_helper](helper_dart)            | Algolia Helpers for Dart    | [![pub package](https://img.shields.io/pub/v/algolia_helper.svg)](https://pub.dev/packages/algolia_helper)                 | alpha version| 
+| [algolia_helper_flutter](helper_flutter) | Algolia Helpers for Flutter | [![pub package](https://img.shields.io/pub/v/algolia_helper_flutter.svg)](https://pub.dev/packages/algolia_helper_flutter) | alpha version | 
```

Comment:
```suggestion
| Package                                  | Description                 | Version                                                                                                                    | Release stage                                                            |
|------------------------------------------|-----------------------------|----------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------|
| [algolia_helper](helper_dart)            | Algolia Helpers for Dart    | [![pub package](https://img.shields.io/pub/v/algolia_helper.svg)](https://pub.dev/packages/algolia_helper)                 | ![alpha version](https://img.shields.io/badge/release-alpha-yellowgreen) |
| [algolia_helper_flutter](helper_flutter) | Algolia Helpers for Flutter | [![pub package](https://img.shields.io/pub/v/algolia_helper_flutter.svg)](https://pub.dev/packages/algolia_helper_flutter) | ![alpha version](https://img.shields.io/badge/release-alpha-yellowgreen) |
```

Maybe we can make it _prettier_ with a badge 🎨

Reply:
Great idea with badges! 👍